examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -242,32 +245,92 @@ const app = {
       }
     );
   },
-  
-  isAnonymous: function() { 
+
+  isAnonymous: function() {
     Purchases.isAnonymous(
       isAnonymous => {
         setStatusLabelText(isAnonymous);
       }
     );
   },
-  
+
   invalidateCustomerInfoCache: function() {
     setStatusLabelText("invalidating customer info cache");
     Purchases.invalidateCustomerInfoCache();
   },
-  
+
   simulatesAskToBuyInSandbox: false,
   toggleSimulatesAskToBuyInSandbox: function() {
     this.simulatesAskToBuyInSandbox = !this.simulatesAskToBuyInSandbox;
     setStatusLabelText("setting simulatesAskToBuyInSandbox to " + this.simulatesAskToBuyInSandbox);
     Purchases.setSimulatesAskToBuyInSandbox(simulatesAskToBuyInSandbox);
   },
-  
+
   presentCodeRedemptionSheet: function() {
     setStatusLabelText("presenting code redemption sheet");
     Purchases.presentCodeRedemptionSheet();
   },
 
+  beginRefundRequestForActiveEntitlement: function() {
+    setStatusLabelText("beginning refund request for active entitlement");
+    Purchases.beginRefundRequestForActiveEntitlement(refundRequestStatus => {
+      setStatusLabelText(Purchases.REFUND_REQUEST_STATUS[refundRequestStatus]);
+    }, error => {
+      setStatusLabelText(error);
+    });
+  },
+
+  beginRefundRequestForEntitlement: function() {
+    setStatusLabelText("beginning refund request by entitlement id");
+    Purchases.getCustomerInfo(
+      customerInfo => {
+        if (!customerInfo.entitlements.active) {
+          setStatusLabelText("customer info doesn't have an active entitlements");
+          return;
+        }
+        let activeEntitlement = Object.values(customerInfo.entitlements.active)[0];
+        setStatusLabelText("beginning refund request for entitlement id: " +
+          activeEntitlement.identifier);
+        Purchases.beginRefundRequestForEntitlement(activeEntitlement, refundRequestStatus => {
+          setStatusLabelText(Purchases.REFUND_REQUEST_STATUS[refundRequestStatus]);
+        }, error => {
+          setStatusLabelText(error);
+        });
+      },
+      error => {
+        setStatusLabelText(error);
+      }
+    );
+  },
+
+  beginRefundRequestForProduct: function() {
+    setStatusLabelText("beginning refund request for product");
```

Comment:
Hmm we should probably mimic what we do for the entitlement version. Maybe 
```suggestion
    setStatusLabelText("beginning refund request by product id");
```
Same below.